classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] What this method is doing right now, is adding the student to the `@studens` variable, but I think you are missing one step in order to achieve the has-many/belongs-to relationship. So here is a sample code that may be helpful to fix this issue:
```
def add_student(student)
  @students << student
  student.classroom = self
end
```
This way, the relationship between classroom and students is properly set. So please work on this issue.